src/pages/signals/components/charts/barchart/tests/BarChart.test.js

```diff
@@ -55,4 +80,14 @@ describe('BarChart Component', () => {
     wrapper.find('Bar').forEach((n) => n.simulate('click'));
     expect(props.onClick).toHaveBeenCalledTimes(3);
   });
+
+  it('should display x reference lines', () => {
+    wrapper.setProps({ xAxisRefLines: [{ x: '2011-01-01', stroke: 'green', strokeWidth: 2 }, { x: '2011-01-03', stroke: 'red', strokeWidth: 2 }]});
+    expect(wrapper).toMatchSnapshot();
```

Comment:
For these two tests, can we snapshot the `ReferenceLine`s only. I think this will work

```suggestion
wrapper.find('ReferenceLine')).forEach((line) => {
  expect(line).toMatchSnapshot();
});
```